ci/get_hdf5_if_needed.sh

```diff
@@ -18,25 +18,28 @@ else
         lib_name=libhdf5.dylib
     else
         lib_name=libhdf5.so
+        ENABLE_DIRECT_VFD="--enable-direct-vfd"
+    fi
+    if [[ "${HDF5_VERSION%.*}" = "1.12" ]]; then
+        ENABLE_BUILD_MODE="--enable-build-mode=production"
     fi
 
-    if [ -f $HDF5_DIR/lib/$lib_name ]; then
-        echo "using cached build"
-    else
+    # if [ -f $HDF5_DIR/lib/$lib_name ]; then
+    #     echo "using cached build"
+    # else
```

Comment:
```suggestion
    if [ -f $HDF5_DIR/lib/$lib_name ]; then
        echo "using cached build"
    else
```

You shouldn't need to comment this out - the cache key in the CI config includes a reference to this script, so if the script changes it won't use the cache:

https://github.com/h5py/h5py/blob/9d625e7270d29275b5610febdc4b0be213871ab7/ci/azure-pipelines-steps.yml#L15

Reply:
Thank you for explaining this to me.